python/activator/activator.py

```diff
@@ -524,7 +524,7 @@ def process_visit(expected_visit: FannedOutVisit):
 
 
 def invalid_visit(e: InvalidVisitError) -> tuple[str, int]:
-    _log.exception()
+    _log.exception(e)
```

Comment:
```suggestion
    _log.exception("Invalid visit: %s", e)
```
?

I think I'd prefer a bit of context rather than just the string text of the exception along with the stack trace.